tests/unit/utils/notification/test_v0.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from amundsen_application import create_app
+from amundsen_application.api.utils.notification_utils import table_key_to_url
+
+local_app = create_app('amundsen_application.config.TestConfig', 'tests/templates')
+
+
+class MetadataTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.mock_table_key = 'db://cluster.schema/table'
+
+    def test_table_key_to_url(self) -> None:
+        """
+        Test successful conversion between key and url
+        :return:
+        """
+        with local_app.app_context():
```

Comment:
Do you actually need to create an application context here? This is simple standalone method so I Imagine something like this would work: 
```
def test_table_key_to_url(self) -> None:
    """
    Test successful conversion between key and url
    :return:
    """
    url = table_key_to_url(table_key=self.mock_table_key
    self.assertEqual('{}/table_detail/cluster/db/schema/table'.format(local_app.config['FRONTEND_BASE']), url)
```
Also note that the assertion is also part of that block as opposed to outside.

Reply:
Spoke offline, and forgot this method needs the app config. Only comment here is to move the assertion into the test.